csrc/quantization/gptq_marlin/gptq_marlin.cu

```diff
@@ -1503,6 +1503,15 @@ bool is_valid_config(thread_config_t const& th_config, int max_m_blocks,
     return false;
   }
 
+  // For act_order with is_k_full == False, we load a chunk of scales.
+  // If GPU L1 cache is small, then reduce the N thread size to ensure
```

Comment:
Can you add a comment about why these constants?

Reply:
Added